stars/default/name.fab

```diff
@@ -1268,13 +1401,16 @@
  11783|76_Cet
  12002|77_Cet
  12093|78_Cet
+ 12048|79_Cet
  12107|80_Cet
  12247|81_Cet
  12387|82_Cet
  12390|83_Cet
  12530|84_Cet
+ 12647|85_Cet*
  12706|86_Cet
  12828|87_Cet
+ 12832|88_Cet*
```

Comment:
Why asterisk added here?

Reply:
According to [38_Ari](https://en.wikipedia.org/wiki/38_Arietis) and [Table of stars with Flamsteed designations](https://en.wikipedia.org/wiki/Table_of_stars_with_Flamsteed_designations)

Reply:
This star hasn’t asterisk in designation